### PR TITLE
[doc] add a tip about using SDXL refiner with hunyuan-dit and pixart

### PR DESCRIPTION
**Pull request by yiyixuxu**

### CODE REVIEW

**Comment by asomoza** [comment]
LGTM thanks!

**Review by asomoza — APPROVED** [review]

File: docs/source/en/using-diffusers/sdxl.md [hunk 1]
```diff
@@ -285,6 +285,12 @@ refiner = DiffusionPipeline.from_pretrained(
 ).to("cuda")
 ```
 
+<Tip warning={true}>
+
+You can also use SDXL refiner with a different base model. For example,you can use[Hunyuan-DiT](https://huggingface.co/docs/diffusers/api/pipelines/hunyuandit) or [PixArt-Sigma](https://huggingface.co/docs/diffusers/api/pipelines/pixart_sigma) Text-to-Image pipelines to generate images with better prompt adherence. Once you have generated an image, you can then pass it to SDXL refiner to enhance final generation quality.
```
Comment (stevhliu) [inline docs/source/en/using-diffusers/sdxl.md:290]:
```suggestion
You can use SDXL refiner with a different base model. For example, you can use the [Hunyuan-DiT](../../api/pipelines/hunyuandit) or [PixArt-Sigma](../../api/pipelines/pixart_sigma) pipelines to generate images with better prompt adherence. Once you have generated an image, you can pass it to the SDXL refiner model to enhance final generation quality.
```

**Review by stevhliu — APPROVED** [review]
LGTM! Just some small suggestions to make it more concise :)

**Comment by vijishmadhavan** [comment]
Hey Can you share inference code? will refiner load along the PixArtSigmaPipeline?

**Pull request merged**